Add support for 2.6 Intree Multitenancy Support (#891)
portworx/pxdocs: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/_index.md

keywords: authorization, security, kubernetes, k8s
---

> This is a reference section describing how to manage Portworx authorization in

Reviewer: ```suggestion
This reference section describes how you can manage Portworx authorization in
```

Kubernetes

{{<info>}}

> For a step by step setup of guide of how to enable Portworx authorization, please see

Reviewer: ```suggestion
**NOTE:** For a step-by-step setup guide of how to enable Portworx authorization, please see
```
